tests/acceptance/steps/helpers/credential-helpers.ts

```diff
@@ -0,0 +1,81 @@
+import { JWTHeaderParameters } from "jose";
```

Comment:
Was hoping to keep helpers out of the step definitions themselves, to keep files smaller/more intentional.